webrtc.html

```diff
@@ -291,14 +291,12 @@
                 Algorithm capabilities of the <a>ICE Agent</a>, and still
                 allows for hash agility as described by [[STUN-BIS]], Section
                 15.3.</p>
-                <div class="issue">[[!RFC7635]] doesn't explicitly define
+                <div class="note">According to [[!RFC7635]] Section 4.1, the
+                HMAC key MUST be a symmetric key. [[!RFC7635]] doesn't explicitly define
```

Comment:
nit: The "key MUST be symmetric" sentence seems out of place here, since the text both above and below is talking about the key shortening. Could just change "issue" to "note" and have 2 notes here.